closures.js

```diff
@@ -151,85 +150,90 @@ motivation('Billy', 'Bob'); // 'Your doing awesome keep it up Billy Bob
 \******************************************************************************/
 
 /****** INSTRUCTIONS PROBLEM 6 ******/
-/* Inside the return create a publicMethod property that is a function that
-invokes privateMethod. After you create the privateMethod. Invoke it by calling
-module.publicMethod(); outside the module scope */
+/* Inside the module's return object create a publicMethod function that
+invokes privateMethod. Invoke this by calling module.publicMethod(); outside
+the module scope */
 
 var module = (function() {
   var person = {
     name: "phillip",
     age: 29,
-    location: 'Utah'
+    location: "Utah"
   };
 
-  var privateMethod = function(){
+  function privateMethod(){
     return "Hi, I'm " + person.name + ", age " + person.age + " from " + person.location;
-  };
+  }
 
   // Anything that is being returned is made public and can be invoked from
-	// outside our lexical scope
-
+  // outside our lexical scope
   return {
     // Code here.
   };
 
 })();
 
-// Uncomment this after you create your public method
-//   module.publicMethod();
 
 
+/******************************************************************************\
+ #PROBLEM-07
+ \******************************************************************************/
+/****** INSTRUCTIONS PROBLEM 7 ******/
+/* Here we are given three arrays: an array of friends, an array of second-level
+friends (friends of friends), and an array of all users. These arrays may share
+users so we must write a function that takes in our existing friends and returns
```

Comment:
Don't pre-fix action statements.  
"These arrays may share users.   Write a function that takes in existing friends and returns a function..."